### DISCUSSION TITLE
Cannot Install Kitty

### DISCUSSION THREAD

**Discussion by edl-xyz** [kovidgoyal/kitty#7155] [Q&A] [answered]
Trying to use kitty on arch linux on my T61 Thinkpad and I get this error when trying to run the command "kitty"

Failed to link GLSL shaders:
error: Too many vertex uniform blocks (1/0)

Every other terminal emulator works except for kitty so I have no clue what is going on.

**Comment by kovidgoyal** [comment]
Your GPU drivers dont support OpenGL 3.3, the minumum required OpenGL
version for kitty, which is over a decade old.

**Reply by edl-xyz** [reply, marked as answer]
Thanks for the point in the right direction. 
I managed to get kitty working by identifying that I had an Intel driver and installing the mesa package. 
After that I had to manually set nomodeset as a parameter in the kernel and rebooted.
Rebooted and now kitty launches!
